### PR TITLE
Add to string()

### PR DESCRIPTION
**Pull request by louis1706**

### CODE REVIEW

**Review by xRoier — CHANGES_REQUESTED** [review]
This should be Role.Type

**Comment by louis1706** [comment]
> https://github.com/Exiled-Team/EXILED/pull/1228/files#diff-bd214a61e3ef03fbb7c16c61869952f84aaf487f81d0066154e399d82668f9beR2362 This should be Role.Type
It's a change of Joker so i will make it only if it's say me yes
https://github.com/Exiled-Team/EXILED/commit/aceb5b19db005c775f25a63027e2fe64295a990a

File: Exiled.API/Features/Roles/Role.cs [hunk 1]
```diff
@@ -146,8 +146,11 @@ public bool Is<T>(out T role)
         /// <inheritdoc/>
         public override bool Equals(object obj) => base.Equals(obj);
 
-        /// <inheritdoc/>
-        public override string ToString() => Type.ToString();
+        /// <summary>
+        /// Returns the role in a human-readable format.
+        /// </summary>
+        /// <returns>A string containing role-related data.</returns>
+        public override string ToString() => $"{Side} {Team} {Type} {IsValid}";
```
Comment (xRoier) [inline Exiled.API/Features/Roles/Role.cs:153]: With this new change you need to change what I told you before in Player::ToString() (```Role.Type``)

Comment (louis1706) [reply Exiled.API/Features/Roles/Role.cs:153]: I prefer it give all this information in Player::ToString

Comment (xRoier) [reply Exiled.API/Features/Roles/Role.cs:153]: Then delete ``{Role?.Team}``

Comment (louis1706) [reply Exiled.API/Features/Roles/Role.cs:153]: Oh yes i have not see it

**Review by xRoier — APPROVED** [review]
Better now, I would only display the Role.Type, not the entire Role info but DevBois have the latest word on this.

**Review by joker-119 — APPROVED** [review]

**Pull request merged**